server/chat-plugins/randombattles/index.ts

```diff
@@ -161,8 +161,9 @@ function getSets(species: string | Species, format: string | Format = 'gen9rando
 	species = dex.species.get(species);
 	const isDoubles = format.gameType === 'doubles';
 	const isBaby = format.team === 'randomBaby';
+	const isCAP = format.team === 'randomCAP';
```

Comment:
It probably isn't working because CAP Random Battle just has `random` for its `team` value.
https://github.com/smogon/pokemon-showdown/blob/26771249c35d7a4b83b8b432d5cb35999166eb23/config/formats.ts#L2502-L2505